### DISCUSSION TITLE
Classic Target Cast Bar

### DISCUSSION THREAD

**Discussion by HavocHank** [parnic/ice-hud#39] [General]
I see the FAQ about classic not having a reliable way of seeing the target cast bar, but I do see a native option for it currently. Perhaps this was added somewhat recently? Is it still not possible to get this feature added to icehud classic?

**Comment by parnic** [comment]
Which version of Classic is this?

**Reply by HavocHank** [reply]
The season of discovery version of classic.

**Reply by parnic** [reply]
Apologies, I don't play Classic. Does that mean Wrath or vanilla?

**Reply by HavocHank** [reply]
Sorry, the vanilla classic. If I understand it right, the vanilla classic version didn't originally come with an interface option to show target cast bars, but now it does.

**Reply by parnic** [reply]
Should be enabled on IceHUD v1.14.30

**Reply by HavocHank** [reply]
That was fast! Thank so much!
